packages/fluent/scss/orgchart/index.scss

```diff
@@ -0,0 +1,33 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "orgchart",
+    dependencies: (
+        "icon",
```

Comment:
I think we should add the avatar as a dependency as well. If you think it's not needed (since it'll be added with the card), you can skip it